playbooks/roles/os_temps/tasks/main.yml

```diff
@@ -37,10 +37,28 @@
   import_tasks: "{{ playbook_dir }}/roles/prereqs/tasks/install_virtual_reqs.yml"
   when: setup_minishift|bool == true
 
-# setup_containers.yml
-- import_tasks: setup_containers.yml
+# Setup project_repo containers
+- include_tasks: "setup_containers.yml os_template_path={{ project_dir }}/{{ os_template_dir }}"
   when: setup_containers|bool == true
 
+# Setup sample_project_repo helper containers
+- include_tasks: "setup_containers.yml os_template_path={{ helper_project_dir }}/{{ helper_os_template_dir }}/helper PARAMS={{ SAMPLE_HELPER_PARAMS }}"
+  when: (setup_containers|bool == true and project_repo != sample_project_repo)
+
+# Check if Jenkins is running
+- name: "Check to see if a Jenkins Master instance is running"
+  shell: "{{ oc_bin }} get pods | grep -i 'running' | grep -i 'jenkins' | tail -1 |  awk '{print $1}'"
```

Comment:
are we in the correct project here for the query?

Reply:
In my testing I was.  Is there something I should do at the top like oc project {{ openshift_project }}

Reply:
I guess a shell task right before would be a safe bet.